main.go

```diff
@@ -30,12 +30,25 @@ func main() {
 		scenariolib.InitLogger(ioutil.Discard, os.Stdout, os.Stdout, os.Stderr)
 	}
 
-	scenariolib.Info.Printf("Queue Length: %v", *queueLength)
-	scenariolib.Info.Printf("Server Port: %v", *port)
+
 
 	source := rand.NewSource(int64(time.Now().Unix()))
 	random := rand.New(source)
 
+	if *queueLength < 1 || *queueLength > 500 {
+		scenariolib.Info.Printf("Queue Length is out of bounds, should be in [1,500].  Set to default value")
+		*queueLength = 100
+	}
+
+	if *routinesPerCPU < 1 || *routinesPerCPU > 5 {
+		scenariolib.Info.Printf("Routine per CPU is out of bounds, should be in [1,5].  Set to default value")
```

Comment:
"Routine per CPU should be in [1,5], will use default value instead."

Similar thing for `Queue Length`